src/EditorFeatures/Core/Implementation/LanguageServer/VisualStudioInProcLanguageServer.cs

```diff
@@ -21,7 +21,7 @@
 
 using LSP = Microsoft.VisualStudio.LanguageServer.Protocol;
 
-namespace Microsoft.VisualStudio.LanguageServices.Implementation.LanguageClient
+namespace Microsoft.CodeAnalysis.Editor.Implementation.LanguageClient
```

Comment:
Does this class need to be renamed?

Reply:
Going to leave as-is since this server is specifically adding the VS LSP extension jsonrpc handling.